File: src/main/java/org/shark/example/domain/user/adapter/outbound/persistence/UserDo.java
package org.shark.example.domain.user.adapter.outbound.persistence;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Table(name = "USER")
@Entity
public class UserDo {
    @Id
    private String id;
    private String name;
    private String email;
}

File: src/main/kotlin/org/shark/example/domain/user/adapter/outbound/persistence/UserDo.kt
package org.shark.example.domain.user.adapter.outbound.persistence

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Table(name = "USER")
@Entity
data class UserDo(
    @Id
    var id: String? = null,
    var name: String? = null,
    var email: String? = null
)
